### PR TITLE
docs: fixes LinkTo and adds new docs to README

### PR DESCRIPTION
**Pull request by dreamwasp**
### Overview

Fixes LinkTo and adds new docs to README. Also adds inline examples to ToolTips after realizing they weren't rendering

### PR Checklist

- [ ] Related to designs:
- [x] Related to JIRA ticket: [GM-917]
- [x] I have run this code to verify it works
- [x] This PR includes unit tests for the code change
- [x] This PR includes testing instructions tests for the code change
- [n/a] The alpha package of this PR is passing end-to-end tests in all relevant Codecademy repositories

#### Testing Instructions

* Check out the new README
* Check some places with LinkTo
* Check out the examples in ToolTips

[GM-917]: https://skillsoftdev.atlassian.net/browse/GM-917?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://676090bf002019600fc02bca--gamut-preview.netlify.app>https://676090bf002019600fc02bca--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/676090bf002019600fc02bca>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://676094b09b9c487c1ba8bb2f--gamut-preview.netlify.app>https://676094b09b9c487c1ba8bb2f--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/676094b09b9c487c1ba8bb2f>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/styleguide@68.3.2-alpha.ffd9de.0<br/></pre>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/styleguide@68.3.2-alpha.8a5e81.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://6760985e000d1117e72e722d--gamut-preview.netlify.app>https://6760985e000d1117e72e722d--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/6760985e000d1117e72e722d>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/styleguide@68.3.2-alpha.3fd558.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://676099c1222b23811bc9678c--gamut-preview.netlify.app>https://676099c1222b23811bc9678c--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/676099c1222b23811bc9678c>Deploy Logs</a></p>

**Review by LinKCoding — COMMENTED** [review]
Thanks for checking through the links!!
I noted some links that didn't work, left some suggestions for consideration, and passed off some QA work that I didn't catch on my own work (sorries). 

Thanks again!!

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/styleguide@68.3.2-alpha.4ac156.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://6761a69db347358b5c35b7c6--gamut-preview.netlify.app>https://6761a69db347358b5c35b7c6--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/6761a69db347358b5c35b7c6>Deploy Logs</a></p>

File: packages/styleguide/src/lib/Molecules/Tips/ToolTip/ToolTip.stories.tsx [hunk 1]
```diff
@@ -14,24 +14,20 @@ const meta: Meta<typeof ToolTip> = {
 export default meta;
 type Story = StoryObj<typeof ToolTip>;
 
-const DefaultExample = () => {
-  return (
+export const Default: Story = {
+  render: () => (
```
[the remark this answers is not in the document]
Comment (dreamwasp) [reply packages/styleguide/src/lib/Molecules/Tips/ToolTip/ToolTip.stories.tsx:18]: np!!! it's inconsistent when is renders and doesn't 🤷

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/styleguide@68.3.2-alpha.92c1da.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://6761b5a2350bc235a75520d2--gamut-preview.netlify.app>https://6761b5a2350bc235a75520d2--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/6761b5a2350bc235a75520d2>Deploy Logs</a></p>

File: packages/styleguide/src/lib/Molecules/Tips/DeprecatedToolTip/DeprecatedToolTip.mdx [hunk 2]
```diff
@@ -18,25 +18,22 @@ export const parameters = {
...
 ### Best practices:
-- Great for surfacing additional information that may or may not be relevant _without_ cluttering the core interface.
 
+- Great for surfacing additional information that may or may not be relevant _without_ cluttering the core interface.
 
 ### When NOT to use:
 
-- If you'd like interactive contents, use a <LinkTo id="molecules-popover">Popover</LinkTo>
-- [use case]- for [describe the use case], use the [similar component] component.
+- If you'd like interactive contents, use a <LinkTo id="Molecules/Popover">Popover</LinkTo>
```
Comment (LinKCoding) [inline packages/styleguide/src/lib/Molecules/Tips/DeprecatedToolTip/DeprecatedToolTip.mdx:36]: missed this before, but should have a period. 🙏

**Review by LinKCoding — APPROVED** [review]
Updates and updated links work great!
I noticed 2 small things that shouldn't take long and are non-blocking :) 
🚀🚀🚀

File: README.md [hunk 3]
```diff
@@ -58,13 +58,13 @@ We provide a single package to manage the versions of a few core dependencies: `
...
+2.  Run `yarn build` to build all of the packages (certain packages like `gamut-icons` need to be built to function in storybook).
 
 ### Running the storybook styleguide
 
-1.  Run `yarn start` to start the storybook server
-1.  Add new stories to `packages/styleguide/stories`
-1.  Stories are written using storybook's [Component Story Format](https://storybook.js.org/docs/formats/component-story-format/)
+1.  Run `yarn nx storybook styleguide` to start the storybook server
+2.  Add new stories to `packages/styleguide/src`
+3.  Stories are written using storybook's [Component Story Format](https://storybook.js.org/docs/formats/component-story-format/) and [MDX](https://storybook.js.org/docs/writing-docs/mdx#basic-example). Check out our Storybook's comprehensive guide on writing stories [here](https://storybook.js.org/docs/react/get-started/introduction).
```
[the remark this answers is not in the document]
Comment (dreamwasp) [reply README.md:67]: i meant to link our storybook 😮‍💨

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/styleguide@68.3.2-alpha.e6422b.0<br/></pre>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/styleguide@68.3.2-alpha.098861.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://6761dc01f655b938571b4a4a--gamut-preview.netlify.app>https://6761dc01f655b938571b4a4a--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/6761dc01f655b938571b4a4a>Deploy Logs</a></p>

**Pull request merged**